Revit_Core_Engine/Convert/Geometry/FromRevit/Point.cs

```diff
@@ -53,9 +53,17 @@ public static oM.Geometry.Point PointFromRevit(this XYZ xyz)
             if (xyz == null)
                 return null;
 
-            return BH.Engine.Geometry.Create.Point(xyz.X.ToSI(SpecTypeId.Length), xyz.Y.ToSI(SpecTypeId.Length), xyz.Z.ToSI(SpecTypeId.Length));
+            return new oM.Geometry.Point { X = xyz.X * m_LengthToSi, Y = xyz.Y * m_LengthToSi, Z = xyz.Z * m_LengthToSi };
         }
 
+
+        /***************************************************/
+        /****              Private fields               ****/
+        /***************************************************/
+
+        // Optimisation to avoid calling ToSI(SpecTypeId.Length) every time as it causes massive overhead
+        private static double m_LengthToSi = 1.0.ToSI(SpecTypeId.Length);
```

Comment:
Would look more consistent...
```suggestion
        private static double m_LengthToSI = 1.0.ToSI(SpecTypeId.Length);
```

Reply:
Not sure if using multiple capitals in a row is the right convention, but we did it so far, so applied the change to keep convention.